litmus-portal/frontend/src/views/ChaosWorkflows/BrowseSchedule/TableData.tsx

```diff
@@ -246,6 +219,88 @@ const TableData: React.FC<TableDataProps> = ({ data, deleteRow }) => {
           </div>
         </Popover>
       </TableCell>
+      <TableCell>
+        <Button
+          onClick={handlePopOverClickForSchedule}
+          style={{ textTransform: 'none' }}
+        >
+          <span
+            className={
+              YAML.parse(data.workflow_manifest).spec.suspend === true
+                ? classes.dark
+                : ''
+            }
+          >
+            <div className={classes.expDiv}>
+              <Typography className={classes.expInfo}>
+                <strong>Show Schedule</strong>
+              </Typography>
+              {isOpenSchedule ? (
+                <KeyboardArrowDownIcon />
+              ) : (
+                <ChevronRightIcon />
+              )}
+            </div>
+          </span>
+        </Button>
+        <Popover
+          id={idSchedule}
+          open={isOpenSchedule}
+          anchorEl={popAnchorElSchedule}
+          onClose={handlePopOverCloseForSchedule}
+          anchorOrigin={{
+            vertical: 'bottom',
+            horizontal: 'center',
+          }}
+          transformOrigin={{
+            vertical: 'top',
+            horizontal: 'center',
+          }}
+          style={{
+            marginTop: 10,
+          }}
```

Comment:
can we use a style class and provide margin with proper units or from the theme.